sympy/integrals/manualintegrate.py

```diff
@@ -458,7 +462,26 @@ def special_function_rule(integral):
                     return p[3](*args)
 
 
-def inverse_trig_rule(integral):
+def _add_degenerate_step(generate_cond, generate_step, degenerate_step):
+    if degenerate_step is None:
+        return generate_step
```

Comment:
Here and elsewhere you are using the word "generate" as the alternative case to the "degenerate" case. I'm not aware of the word "generate" being used in this way. Usually the alternative to "degenerate" is "generic". These words have different meanings in Mathematics than they do in ordinary English but in any case as far as I know "generate" is always a verb whereas "degenerate" and "generic" are usually adjectives.